packages/cli/src/telemetry/index.ts

```diff
@@ -43,11 +43,11 @@ export default {
     const telemetryOptions = await checkOptIn(interactive);
     if (telemetryOptions === undefined || !telemetryOptions.optIn) return;
 
-    // extract network name if present
-    let network;
-    if ('network' in options) {
-      network = options.network;
-      if (network.match(/dev-/)) network = 'development';
+    // normalize network name
+    const { ZWeb3 } = await import('@openzeppelin/upgrades');
+    let network = await ZWeb3.getNetworkName();
```

Comment:
This normalization was being handled in ConfigManager#initNetworkConfiguration, which means that all calls after the initNetworkConfiguration got the normalized network name. If we need to do this at the telemetry level, this _might_ mean that we are failing to normalize at the command level. Can you double check?